src/init.cpp

```diff
@@ -1790,11 +1801,34 @@ bool AppInitMain(NodeContext& node, interfaces::BlockAndHeaderTipInfo* tip_info)
     bool do_reindex{args.GetBoolArg("-reindex", false)};
     const bool do_reindex_chainstate{args.GetBoolArg("-reindex-chainstate", false)};
 
+    std::optional<swiftsync::HintsfileReader> utxo_hints;
+    if (args.IsArgSet("-utxohints")) {
+        fs::path path = fs::absolute(args.GetPathArg("-utxohints"));
+        if (!fs::exists(path)) {
```

Comment:
We could exercise these errors (file doesn't exist and open failure) in the functional tests. An (obvious) unkilled mutant:

```diff
diff --git a/src/init.cpp b/muts-pr-34004-init-cpp/init.mutant.1.cpp
index fb7336f9ae..59de0f6b7a 100644
--- a/src/init.cpp
+++ b/muts-pr-34004-init-cpp/init.mutant.1.cpp
@@ -1812,7 +1812,7 @@ bool AppInitMain(NodeContext& node, interfaces::BlockAndHeaderTipInfo* tip_info)
         AutoFile afile{file};
         if (afile.IsNull()) {
             LogError("Failed to open UTXO hint file.");
-            return false;
+            return true;
         }

```